spec/hackerone/client/reporter_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+
+RSpec.describe HackerOne::Client::Report do
+  let(:api) { HackerOne::Client::Api.new("github") }
+
+  before(:all) do
+    ENV["HACKERONE_TOKEN_NAME"] = "foo"
+    ENV["HACKERONE_TOKEN"] = "bar"
+  end
+
+  let(:reporters) do
+    VCR.use_cassette(:reporters) do
+      api.reporters
+    end
+  end
+
+  let(:reporter) do
+    reporters.first
+  end
+
+  it "returns a collection" do
+    expect(reporters).to be_kind_of(Array)
+    expect(reporters.size).to eq 2
+  end
+
+  it "returns id" do
+    expect(reporter.id).to be_present
+    expect(reporter.id).to eq '3683'
+  end
+
+  it "returns disabled?" do
+    expect(reporter.disabled?).to eq false
```

Comment:
I have no idea if this follows any style guide, but I like to wrap the argument to the matchers in parens (but not the args to `to`). e.g.

```ruby
expect(reporter.disabled?).to eq(false)
```